build-tools/create-packs/Microsoft.Android.Ref.proj

```diff
@@ -36,10 +34,18 @@ by projects that use the Microsoft.Android framework in .NET 5.
     </PropertyGroup>
 
     <ItemGroup>
-      <_PackageFiles Include="@(_AndroidAppRefAssemblies)" PackagePath="$(_AndroidRefPackAssemblyPath)" TargetPath="$(_AndroidRefPackAssemblyPath)" />
-      <_PackageFiles Include="$(XAInstallPrefix)xbuild-frameworks\Microsoft.Android\netcoreapp3.1\mono.android.jar" PackagePath="$(_AndroidRefPackAssemblyPath)" />
-      <_PackageFiles Include="$(XAInstallPrefix)xbuild-frameworks\Microsoft.Android\netcoreapp3.1\mono.android.dex" PackagePath="$(_AndroidRefPackAssemblyPath)" />
-      <_PackageFiles Include="$(XAInstallPrefix)xbuild-frameworks\Microsoft.Android\netcoreapp3.1\AndroidApiInfo.xml" PackagePath="$(_AndroidRefPackAssemblyPath)" />
+      <!-- NOTE: we don't have a reference assembly for Java.Interop.dll yet -->
+      <_AndroidRefPackAssemblies Include="$(_MonoAndroidNETOutputDir)Java.Interop.dll" />
```

Comment:
You might check if we have this file now since we bumped Java.Interop:

https://github.com/xamarin/java.interop/commit/6cde08775fbec5506c02c69857df0f0f75995c94

Reply:
I think we may need to copy it from `external/Java.Interop/bin/$(Configuration)-netcoreapp3.1/ref/Java.Interop.dll`, it doesn't look like it's automatically copied to the Mono.Android output dir. Maybe there is some msbuild property that controls ref assembly copying for project references that I can find.

Reply:
I updated this file to use the external JI path, I couldn't find a built in way to copy this over to the Mono.Android netcore output directory.